pom.xml

```
@@ -82,7 +82,7 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins.workflow</groupId>
             <artifactId>workflow-api</artifactId>
-            <version>2.32</version>
+            <version>2.34-rc895.9baeb8950155</version>
```

API half was released in 2.36.

```suggestion
            <version>2.36</version>
```

---

Folded into the existing commit